api/models/test.js

```diff
@@ -7,6 +7,7 @@ let mixin = require('./modelMixin.js');
 let model = Object.assign({
 	title: 'string',
 	questions: [{ref:'Question', type: 'ObjectId'}],
+	hidden: 'string',
```

Comment:
is the default value defined here, like:
`hidden: false`

or does `string` refer to the type of value the model should accept?